tests/Bootstrap/MockLogger.php

```diff
@@ -4,15 +4,59 @@
 
 namespace MyParcelNL\Pdk\Tests\Bootstrap;
 
+use MyParcelNL\Pdk\Base\FileSystemInterface;
+use MyParcelNL\Pdk\Facade\Pdk;
 use MyParcelNL\Pdk\Logger\AbstractLogger;
+use Psr\Log\LogLevel;
+use function array_filter;
+use function array_reduce;
+use function json_encode;
 
 class MockLogger extends AbstractLogger
 {
+    private const ALL_LOG_LEVELS = [
+        LogLevel::ALERT,
+        LogLevel::CRITICAL,
+        LogLevel::DEBUG,
+        LogLevel::EMERGENCY,
+        LogLevel::ERROR,
+        LogLevel::INFO,
+        LogLevel::NOTICE,
+        LogLevel::WARNING,
+    ];
+
+    /**
+     * @var \MyParcelNL\Pdk\Base\FileSystemInterface
+     */
+    private $fileSystem;
+
     /**
      * @var array
      */
     private $logs = [];
 
+    /**
+     * @var array<string, resource>
+     */
+    private $streams = [];
+
+    /**
+     * @param  \MyParcelNL\Pdk\Base\FileSystemInterface $fileSystem
+     */
+    public function __construct(FileSystemInterface $fileSystem)
+    {
+        $this->fileSystem = $fileSystem;
+
+        $appInfo = Pdk::getAppInfo();
+        $this->fileSystem->mkdir($appInfo->createPath('logs'), true);
+
+        foreach (self::ALL_LOG_LEVELS as $level) {
+            $filename = $appInfo->createPath("logs/test_$level.log");
```

Comment:
Ik denk dat dit de oorzaak is van de volgende error aan het einde van het draaien van de testsuite;

```
rmdir(.tmp//logs): Directory not empty
```

Reply:
ja en nee, het is een probleem in ongerelateerde test suite setup code, maar wordt nu pas zichtbaar door deze toevoeging, ik zal het oplossen in main